src/packages/@ncigdc/containers/explore/CaseTable.js

```diff
@@ -137,67 +130,37 @@ export const CaseTableComponent = compose(
         />
       </Row>
       <div style={{ overflowX: "auto" }}>
-        <table style={styles.table} id="explore-case-table">
-          <thead>
-            <Tr>
-              <Th rowSpan="2">Case UUID</Th>
-              <Th rowSpan="2">Submitter ID</Th>
-              <Th rowSpan="2">Project</Th>
-              <Th rowSpan="2">Primary Site</Th>
-              <Th rowSpan="2">Gender</Th>
-              <Th rowSpan="2" style={styles.right}>Files</Th>
-              <Th
-                colSpan={Object.entries(DATA_CATEGORIES).length}
-                style={styles.center}
-              >
-                Available Files per Data Category
-              </Th>
-              <Th rowSpan="2" style={styles.right}>
-                <Tooltip
-                  Component="# Simple Somatic Mutations Filtered by current criteria"
-                  style={tableToolTipHint()}
-                >
-                  # Mutations
-                </Tooltip>
-              </Th>
-              <Th rowSpan="2" style={styles.right}>
-                <Tooltip
-                  Component="# Genes with Simple Somatic Mutations Filtered by current criteria"
-                  style={tableToolTipHint()}
-                >
-                  # Genes
-                </Tooltip>
-              </Th>
-            </Tr>
-
-            <Tr>
-              {Object.values(DATA_CATEGORIES).map((category: any) => (
-                <Th key={category.abbr} style={styles.right}>
-                  <abbr>
-                    <Tooltip
-                      Component={category.full}
-                      style={tableToolTipHint()}
-                    >
-                      {category.abbr}
-                    </Tooltip>
-                  </abbr>
-                </Th>
+        <Table
+          id="repository-case-table"
```

Comment:
match these ids https://github.com/NCI-GDC/portal-ui/pull/1775/files#diff-8fcaeb97633b0ed933b0b7207117e714L135 so the tsv table download works